rest/utilities_testing_blip_client.go

```diff
@@ -78,6 +78,30 @@ type BlipTesterClient struct {
 
 	collectionClients        []*BlipTesterCollectionClient
 	nonCollectionAwareClient *BlipTesterCollectionClient
+
+	hlc *hybridLogicalClock
+}
+
+// hybridLogicalClock produces a UnixNano timestamp that will always be increasing regardless of clock changes.
```

Comment:
Could we pull `hybridLogicalClock` out of rosmar and into `sg-bucket` for use by both?

https://github.com/couchbaselabs/rosmar/blob/main/hlc.go